common/src/main/java/org/apache/celeborn/common/write/SimplePushStrategy.java

```diff
@@ -53,6 +53,6 @@ public void limitPushSpeed(PushState pushState, String hostAndPushPort) throws I
 
   @Override
   public int getCurrentMaxReqsInFlight(String hostAndPushPort) {
```

Comment:
the method name still confusing

Reply:
> the method name still confusing

@pan3793 This method means to get the max in-flight request number to a specific worker for worker-level traffic control. The parameter is the address of a specific worker's address, so it's not necessary to add ```perWorker``` in the signature.